src/server/routes/units.js

```diff
@@ -1,258 +1,146 @@
+// routes/units.js
 /* This Source Code Form is subject to the terms of the Mozilla Public
-  * License, v. 2.0. If a copy of the MPL was not distributed with this
-  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-const express = require('express');
-const { log } = require('../log');
-const { getConnection } = require('../db');
-const Unit = require('../models/Unit');
-const { removeAdditionalConversionsAndUnits } = require('../services/graph/handleSuffixUnits');
-const validate = require('jsonschema').validate;
-const { success, failure } = require('./response');
-
-const router = express.Router();
-
-function formatUnitForResponse(item) {
-	return {
-		id: item.id, name: item.name, identifier: item.identifier, unitRepresent: item.unitRepresent,
-		secInRate: item.secInRate, typeOfUnit: item.typeOfUnit, suffix: item.suffix,
-		displayable: item.displayable, preferredDisplay: item.preferredDisplay, note: item.note,
-		minVal: item.minVal, maxVal: item.maxVal, disableChecks: item.disableChecks
-	};
-}
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+ const express = require('express');
+ const { authMiddleware } = require('./authenticator');
+ const { log } = require('../log');
+ const { getConnection } = require('../db');
+ const Unit = require('../models/Unit');
+ const { removeAdditionalConversionsAndUnits } = require('../services/graph/handleSuffixUnits');
+ const validate = require('jsonschema').validate;
+ const { success, failure } = require('./response');
+ 
+ const router = express.Router();
+ 
 /**
- * Route for getting all units.
+ * Route for listing all units.
  */
-router.get('/', async (req, res) => {
-	const conn = getConnection();
-	try {
-		const rows = await Unit.getAll(conn);
-		res.json(rows.map(formatUnitForResponse));
-	} catch (err) {
-		log.error(`Error while performing GET units details query: ${err}`, err);
-	}
-});
-
+ router.get(
```

Comment:
The formatting was changed so this is now multiline. It does not match other files and causes more indenting. Could it please be done the other way.

Also, the indenting is now with spaces instead of tabs. VSC an change this throughout the file. Ask if you need help. Note that here there should be no indenting here,